src/GitHub.App/ViewModels/PullRequestListViewModel.cs

```diff
@@ -269,6 +276,8 @@ public IAccount EmptyUser
         public ReactiveCommand<object> OpenPullRequest { get; }
         public ReactiveCommand<object> CreatePullRequest { get; }
 
+        public ReactiveCommand<object> OpenPROnGitHub { get; }
```

Comment:
If we're moving things about here, might it be worth making the naming more consistent, i.e. "OpenPullRequestOnGitHub`?